### PR TITLE
Feature/stefan concat input

### PR DESCRIPTION
**Pull request by lslivins**
Refactor predict_bias.py for consistency with Sergey's modifications to create the input files and neural networks. Some generality added (e.g. where to save the output increment), but might need a bit more (e.g. the "sample increment" file currently points to Laura's directory).

Updated code has been tested and runs successfully.

### CODE REVIEW

File: predict_bias.py [hunk 1]
```diff
@@ -71,45 +78,49 @@ def read_input(): # needs clean up with the proprocessor and dataset
...
     
     ins = torch.cat([torch.from_numpy(np.concatenate(vals_f, axis=0)[None,]), 
-                     torch.from_numpy(np.concatenate(sfcs, axis=0)[None,slice_sfc])], 1).float()
+                     torch.from_numpy(np.concatenate(sfcs, axis=0)[None,])], 1).float()
+    print(np.shape(torch.from_numpy(np.concatenate(vals_f, axis=0)[None,])))
+    print(np.shape(torch.from_numpy(np.concatenate(sfcs, axis=0)[None,])))
     # Prepare Normalizing mean and std
-    ddd='/scratch2/NCEPDEV/stmp1/Tse-chun.Chen/anal_inc/npys/ifs'
-    mean_f06 = torch.from_numpy(np.load(ddd+'_f06_ranl_{}_mean_1d.npy'.format(trunc))[slice_f06])
+    ddd='/scratch2/BMC/gsienkf/Sergey.Frolov/fromStefan/npys_sergey2/ifs'
```
Comment (frolovsa) [inline predict_bias.py:99]: Should we move this line above function declaration so it is specified once for this script?

File: predict_bias.py [hunk 2]
```diff
@@ -188,8 +192,8 @@ def column(var):
 
 if inc_sfg == 'output':
     ## sample file:
-    file_i = xr.open_dataset('/scratch1/NCEPDEV/stmp2/Tse-chun.Chen/anal_inc/IFS_replay/2020010100/control/INPUT/fv3_increment6.nc')
-    logging.info("saving to fv3_increment6.nc")
+    file_i = xr.open_dataset('/scratch2/BMC/gsienkf/Laura.Slivinski/model_error_corr_work/%s/fv3_increment6.nc'%indate) #later, make this more general
```
Comment (frolovsa) [inline predict_bias.py:195]: can we pass the output dir on the command line? we can make `./` to be the default.

File: predict_bias.py [hunk 3]
```diff
@@ -137,15 +148,13 @@ def column(var):
 if len(sys.argv) > 5:
     file_f = xr.open_dataset(sys.argv[5])
```
Comment (frolovsa) [inline predict_bias.py:149]: can we add comments for what argv[5,6,7] are?

**Review by frolovsa — CHANGES_REQUESTED** [review]
Hi, Thank you for working on this. Looks really good. i just have a couple of minor clean up comments.

File: predict_bias.py (hunk 3 above)
Comment (lslivins) [reply predict_bias.py:149]: Yes, I can do this.

File: predict_bias.py (hunk 2 above)
Comment (lslivins) [reply predict_bias.py:195]: "file_i" is not the output dir, I did add the output dir as a command line argument. "file_i" is the "sample" increment that the code takes to generate all the variables and metadata before copying it to a new increment file and writing over the values. I could make this a separate command line argument though (just starts to be a very long list of long directory arguments).

File: predict_bias.py (hunk 1 above)
Comment (lslivins) [reply predict_bias.py:99]: Sure I'll do that

File: predict_bias.py (hunk 2 above)
Comment (frolovsa) [reply predict_bias.py:195]: right. can we assume that fv3_increment6.nc is located in the output directory?

Comment (lslivins) [reply predict_bias.py:195]: Oh I see. Sure.

**Review by frolovsa — APPROVED** [review]
looks good. thank you. I am going to go ahead and merge

**Pull request merged**